GiniVision/Classes/Core/DocumentPickerCoordinator.swift

```diff
@@ -227,10 +234,10 @@ extension DocumentPickerCoordinator: UIDropInteractionDelegate {
         }
     }
     
-    fileprivate func loadDocuments<T: NSItemProviderReading>(ofClass classs: T.Type,
-                                                             from session: UIDropSession,
-                                                             in group: DispatchGroup,
-                                                             completion: @escaping (([T]?) -> Void)) {
+    private func loadDocuments<T: NSItemProviderReading>(ofClass classs: T.Type,
```

Comment:
Why not `fileprivate` anymore?

Reply:
Because I want to make it visible only for the extension (Drag and drop related stuff).